src/Microsoft.AspNet.Mvc.DataAnnotations/ModelClientValidationNumericRule.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// Represents client side validation rule for numeric values.
```

Comment:
`Represents` is a dirty word. This _is_ a `<see cref="ModelClientValidationRule"/>` for numeric values.